src/views/GroupSettings.vue

```diff
@@ -75,8 +75,10 @@
 
 <script>
 import axios from 'nextcloud-axios'
-import { Multiselect } from 'nextcloud-vue'
-import _ from 'lodash'
+import debounce from 'debounce'
+import Multiselect from 'nextcloud-vue/dist/Components/Multiselect'
+import { linkToOCS } from 'nextcloud-router'
```

Comment:
I think you should really test this https://github.com/ChristophWurst/nextcloud-router/blob/166d5622d53c49290556d6e48b1338ae25085bb7/lib/index.ts#L37 :see_no_evil:

Reply:
Done! :D